data/campaigns/tutorial/gui/character_selection.cfg

```diff
@@ -39,17 +39,21 @@
                             grow_factor = 1
                             border = "all"
                             border_size = 5
+                            # wmllint: markcheck off
```

Comment:
Seems a bug in wmllint, if it's not recognising these as wanting image filenames.

Reply:
Does someone need to create an issue for that?

Reply:
Good idea, logged it as #6995.